lapis/nginx.lua

```diff
@@ -109,7 +109,8 @@ local ngx_req = {
   end,
   parsed_url = function(t)
     local uri = ngx.var.request_uri
-    uri = uri:match("(.-)%?") or uri
+    local pos = uri:find("?")
```

Comment:
probably an insignificant amount, there is no pattern on the single character pattern, so it will still end up matching it literally. Regarding combining the two lines, I don't really see an advantage to making it harder to read